policies/pull-request-review-and-merge.md

```diff
@@ -0,0 +1,65 @@
+# Policy on pull request review and merging in the Astropy Project
+
+## Astropy core
+
+In the astropy core package the requirements are as follows:
+
+- At least one contributor with relevant expertise must review the pull request
+  in detail.
+  - New or updated code is logically correct and achieves the desired effect of
```

Comment:
```suggestion
  - New or updated code is correct and achieves the desired effect of
```
I'm not sure what "logically" meant here, but I don't really think it's needed anyway!